eng/SourceBuild.props

```diff
@@ -1,3 +1,5 @@
+<!-- Whenever altering this or other Source Build files, please include @dotnet/source-build-internal as a reviewer. -->
```

Comment:
If this is for file-level review perhaps we could use GH owners file?

Reply:
> If this is for file-level review perhaps we could use GH owners file?

Initially wanted to add both the comment and the `CODEOWNERS` entry, but for a team to be added as a code owner it needs to have [write permissions](https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/customizing-your-repository/about-code-owners#example-of-a-codeowners-file) to the repo and I wanted to verify first if this is something we would need. I should've mentioned that in the description though, my bad, forgot to edit it.

If you are OK with granting write permissions to `dotnet/source-build-internal`, I will add the `CODEOWNERS` entry.

Reply:
Hrm, probably not.

Reply:
Then unfortunately the comments are our best bets.

We will monitor how effective they are and will of course change / adapt based on performance. But for now this is a relatively cheap / easy option

Reply:
I'd be fine with giving dotnet/source-build-internal `write` access - we already have 500+ users with that level of access (everyone in dotnet/npt). Any objections @dotnet/aspnet-admins?

Reply:
@danmoseley, @mkArtak any opposition to giving dotnet/sorce-build-internal `write` access to the repo so that we can add them to `Codeowners` for this file?

Reply:
> @danmoseley, @mkArtak any opposition to giving dotnet/sorce-build-internal write access to the repo so that we can add them to Codeowners for this file?

if the owners of dotnet/source-build-internal believe that is reasonable, sure

Reply:
@oleksandr-didyk I've given dotnet/source-build-internal write access to the repo - feel free to update the PR to add the to `CODEOWNERS` for this file

Reply:
Thank you, added the entry for the file

As for altering the comment to make it more visible (discussed below) - given the valid CODEOWNERS entry there is less pressure on the comment so I opted to keeping it the same as in other repositories